File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F16.P1.cpp
#include <cstdint>
#include <cstdio>
#include <numeric>
#include <vector>

std::int32_t sum_sequence() {
// BEGIN PROBE F16.P1
std::vector<std::int32_t> xs{1, 2, 3};
std::int32_t total = std::reduce(xs.begin(), xs.end());
return total;
// END PROBE F16.P1
}

int main() {
    std::printf("%d\n", sum_sequence());
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F16.P1.java
import java.util.ArrayList;
import java.util.List;

class Main {
    static int sumSequence() {
        // BEGIN PROBE F16.P1
        var xs = new ArrayList<>(List.of(1, 2, 3));
        int total = xs.stream().reduce(0, Integer::sum);
        return total;
        // END PROBE F16.P1
    }

    public static void main(String[] args) {
        System.out.println(sumSequence());
    }
}
